### PR TITLE
Some signin/signup polish and some warnings/assertion tidying.

### PR DESCRIPTION
**Pull request by adamvy**
- Hide OIDC/Google Signin in sign up view
- Fix typo in default CSP

- Fix a bunch of assertion failures about missing imports
  - Remove some unused imports
  - Mark some imports as optional as they're not always present
 
- Make copyFromValue a property in base class to remove warning about change of axiom type
- Make PasswordPolicy.validate context oriented as it's not always instantiated in a context with "commonPasswordDAO" 
  - For example, during sign in the default policy is instantated under the theme, but this is not in a subcontext on client

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]
Nice to have you back.

File: src/foam/nanos/cron/CronSchedule.js [hunk 1]
```diff
@@ -223,7 +223,9 @@ foam.CLASS({
     },
     {
       class: 'Array',
-      of: 'Int',
+      // TODO(adamvy): Should be of: 'Int', but when IntArray was added to types it broke this
+      // and I haven't figured out why yet.
+      of: 'Object',
```
Comment (kgrgreer) [inline src/foam/nanos/cron/CronSchedule.js]: I fixed this. Can you revert?

File: src/foam/nanos/cron/SimpleIntervalSchedule.js [hunk 2]
```diff
@@ -197,7 +197,9 @@ foam.CLASS({
     },
     {
       class: 'Array',
-      of: 'Int',
+      // TODO(adamvy): Should be of: 'Int', but when IntArray was added to types it broke this
+      // and I haven't figured out why yet.
+      of: 'Object',
```
Comment (kgrgreer) [inline src/foam/nanos/cron/SimpleIntervalSchedule.js]: I fixed this. Can you revert?

Comment (adamvy) [reply src/foam/nanos/cron/SimpleIntervalSchedule.js]: done

File: src/foam/nanos/cron/CronSchedule.js (hunk 1 above)
Comment (adamvy) [reply src/foam/nanos/cron/CronSchedule.js]: done

**Pull request merged**